assets/schema_input.json

```diff
@@ -10,27 +10,30 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["id"]
             },
             "fastq_1": {
                 "type": "string",
+                "format": "file-path",
+                "exists": true,
                 "pattern": "^\\S+\\.f(ast)?q\\.gz$",
                 "errorMessage": "FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
             },
             "fastq_2": {
-                "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'",
-                "anyOf": [
-                    {
-                        "type": "string",
-                        "pattern": "^\\S+\\.f(ast)?q\\.gz$"
-                    },
-                    {
-                        "type": "string",
-                        "maxLength": 0
-                    }
-                ]
+                "type": "string",
+                "format": "file-path",
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            },
+            "fastq_3": {
+                "type": "string",
+                "format": "file-path",
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 3 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
             }
-        },
-        "required": ["sample", "fastq_1"]
-    }
+        }
+    },
+    "required": ["sample"],
+    "anyOf": [{ "required": ["fastq_1"] }, { "required": ["fastq_3"] }]
```

Comment:
How is one suppose to understand that? Does it mean that `fastq_1` or `fastq_3` should be in the samplesheet?

Reply:
Right, `fastq_1` would be for single-end (or paired-end) short reads, `fastq_3` for single-end long reads

Reply:
@mirpedrol : Why isn't the syntax just `"anyOf": ["fastq_1", "fastq_3"]`?

Reply:
agree with @asp8200 on the samplesheet. what is the reason for using fastq1 and fastq3 for R1 and R2?

Reply:
I agree with you that the wording is a bit misleading, will change that 👍 
to be able to run the pipeline with either short reads or long reads either fastq_1 or fastq_3 must be present.

Reply:
Is there a parameter which I have to set in the `schema_input.json` to throw an error if only `short_reads_fastq_2` is present in the `samplesheet.csv`? have adapted the naming and now i wanted to remove the check mentioned here https://github.com/nf-core/detaxizer/pull/22#discussion_r1528419331 and use `nf-validation` for it. 
So far, the pipeline is running only with correct input in the `samplesheet.csv` but does not throw an error otherwise and runs empty...